content/en/observability_pipelines/setup/_index.md

```diff
@@ -393,6 +393,29 @@ For a deeper dive into recommendations for deploying and scaling multiple Worker
 - See [Deployment Design and Principles][7] for information on what to consider when designing your Observability Pipelines architecture.
 - See [Best Practices for OPW Aggregator Architecture][8] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.
 
+## Deployment Modes
+
+<div class="alert alert-warning">
+  Remote configuration for Observability Pipelines is in private beta. Contact <a href="https://docs.datadoghq.com/help/">Datadog support</a> or your Customer Success Manager for access.
+</div>
+If you are enrolled in the private beta of [Remote Configuration][7] to remotely roll out changes to your OP Workers, you remotely roll out changes to your Workers from the Datadog UI, rather than making updates to your pipeline configuration in a text editor and rolling out your changes manually. You can choose your deployment method when you create a pipeline and install your Workers.
```

Comment:
```suggestion
If you are enrolled in the private beta of [Remote Configuration][9], you can remotely roll out changes to your Workers from the Datadog UI, rather than make updates to your pipeline configuration in a text editor and then manually rolling out your changes. Choose your deployment method when you create a pipeline and install your Workers.
```